virttest/env_process.py

```diff
@@ -1096,6 +1113,25 @@ def postprocess(test, params, env):
             sosreport_path = vm.sosreport()
             logging.info("Sosreport for guest: %s", sosreport_path)
 
+    # Collect code coverage report for qemu if enabled
+    if params.get("gcov_qemu", "no") == "yes":
+        qemu_builddir = os.path.join(test.bindir, "build", "qemu")
+        if os.path.isdir(qemu_builddir) and utils_package.package_install("gcovr"):
+            gcov_qemu_dir = utils_misc.get_path(test.debugdir, "gcov_qemu")
+            os.makedirs(gcov_qemu_dir)
+            os.chdir(qemu_builddir)
+            collect_cmd_opts = params.get("gcov_qemu_collect_cmd_opts", "--html")
+            collect_cmd = "gcovr -j %s -o %s -s %s ." % (cpu_utils.online_cpus_count(),
```

Comment:
@sathnaga How you are handling when the binary `gcovr` is not installed or not available in the `PATH`.

Reply:
ah, okies, I expect as this is standard utility it should be available in PATH, let me see if I can handle it better ...

Reply:
As `gcovr` gets installed, we are guaranteed, gcovr commandline would work, so PATH handle might not need.